simulator-spring-boot/src/main/java/org/citrusframework/simulator/config/SimulatorConfigurationProperties.java

```diff
@@ -114,4 +116,15 @@ public void setEnvironment(Environment environment) {
     public void afterPropertiesSet() {
         logger.info("Using the simulator configuration: {}", this);
     }
+
+    @Getter
+    @Setter
+    @ToString
+    public static class TestResults {
```

Comment:
you're right.. kinda. I'll rename it to `SimulationResults`. from a customers perspective (without exact knowledge of the inner workings) this would also make more sense.